darwin/future/meta/objects/base.py

```diff
@@ -21,4 +21,44 @@ def __init__(
         self.meta_params = meta_params or {}
 
     def __str__(self) -> str:
+        class_name = self.__class__.__name__
+        if class_name == "Team":
+            return f"Team\n\
+- Team Name: {self._element.name}\n\
+- Team Slug: {self._element.slug}\n\
+- Team ID: {self._element.id}\n\
+- {len(self._element.members if self._element.members else [])} member(s)"
+
+        elif class_name == "TeamMember":
+            return f"Team Member\n\
+- Name: {self._element.first_name} {self._element.last_name}\n\
+- Role: {self._element.role.value}\n\
+- Email: {self._element.email}\n\
+- User ID: {self._element.user_id}"
+
+        elif class_name == "Dataset":
+            releases = self._element.releases
+            return f"Dataset\n\
+- Name: {self._element.name}\n\
+- Dataset Slug: {self._element.slug}\n\
+- Dataset ID: {self._element.id}\n\
+- Dataset Releases: {releases if releases else 'No releases'}"
+
+        elif class_name == "Workflow":
+            return f"Workflow\n\
+- Workflow Name: {self._element.name}\n\
+- Workflow ID: {self._element.id}\n\
+- Connected Dataset ID: {self._element.dataset.id}\n\
+- Conneted Dataset Name: {self._element.dataset.name}"
+
+        elif class_name == "Stage":
+            return f"Stage\n\
+- Stage Name: {self._element.name}\n\
+- Stage Type: {self._element.type.value}\n\
+- Stage ID: {self._element.id}"
+
+        else:
+            raise Exception(f"Class name {class_name} not found in __str__ method")
```

Comment:
Might be useful to add a `pprint` of anything that makes it to this point.  I'm on the fence whether this should raise, because the most likely case of this happenning is if we forget to implement a new one.  I guess we'd find out quickly, though.

Reply:
Good point, I nearly didn't include this.

Will just pprint anything that reaches it w/out raising